src/prefect/cli/orion.py

```diff
@@ -138,22 +139,33 @@ async def start(
             )
         )
 
-        # Explicitly handle the interrupt signal here, as it will allow us to cleanly
-        # stop the Orion uvicorn server with a SIGTERM.  Failing to do that may cause
-        # a large amount of anyio error traces on the terminal, because the SIGINT is
-        # handled by Typer/Click in this (the parent process) and will start
-        # shutting down subprocesses:
+        # Explicitly handle the interrupt signal here, as it will allow us to
+        # cleanly stop the Orion uvicorn server with a SIGTERM. Failing to do
+        # that may cause a large amount of anyio error traces on the terminal,
+        # because the SIGINT is handled by Typer/Click in this (the parent
+        # process) and will start shutting down subprocesses:
         # https://github.com/PrefectHQ/orion/issues/2475
-        # The first interrupt with send a SIGTERM to uvicorn, then subsequent ones will
-        # send SIGKILL
-        def stop_orion(*args):
-            app.console.print("\nStopping Orion...")
-            os.kill(orion_process_id, signal.SIGTERM)
-            signal.signal(signal.SIGINT, kill_orion)
-
-        def kill_orion(*args):
-            app.console.print("\nKilling Orion...")
-            os.kill(orion_process_id, signal.SIGKILL)
+
+        # First interrupt with send a SIGTERM to uvicorn, then subsequent ones
+        # will send SIGKILL. In Windows we use CTRL_BREAK_EVENT as SIGTERM is
+        # useless (https://bugs.python.org/issue26350).
+
+        if sys.platform == "win32":
+
+            def stop_orion(*args):
+                app.console.print("\nStopping Orion...")
+                os.kill(orion_process_id, signal.CTRL_BREAK_EVENT)
+
+        else:
+
+            def stop_orion(*args):
+                app.console.print("\nStopping Orion...")
+                os.kill(orion_process_id, signal.SIGTERM)
+                signal.signal(signal.SIGINT, kill_orion)
+
+            def kill_orion(*args):
+                app.console.print("\nKilling Orion...")
+                os.kill(orion_process_id, signal.SIGKILL)
```

Comment:
This patterns seems useful as a `send_kill_signal(pid: int)` utility in `processutils` so future consumers of this pattern don't need to worry about Windows specific logic.

Reply:
(It'd also make it easier to test)

Reply:
Good idea! I'll work down that route.